WireDomain/Sources/WireDomain/Providers/InitialSyncProvider.swift

```diff
@@ -18,14 +18,9 @@
 
 import Foundation
 
-// sourcery: AutoMockable
```

Comment:
question: not needed anymore?

Reply:
no, in the end the test case itself could conform to the provider protocols.